enginetest/queries/queries.go

```diff
@@ -4213,6 +4213,14 @@ Select * from (
 		Query:    "SELECT id FROM typestable WHERE da < date_sub('2020-01-01', INTERVAL 1 DAY)",
 		Expected: nil,
 	},
+	{
+		Query:    "SELECT id FROM typestable WHERE da >= subdate('2020-01-01', INTERVAL 1 DAY)",
+		Expected: []sql.Row{{int64(1)}},
+	},
+	{
+		Query:    "SELECT id FROM typestable WHERE da >= subdate('2020-01-01', 1)",
+		Expected: []sql.Row{{int64(1)}},
+	},
```

Comment:
Can you add enginetests that use columns from a table
Something like:
```sql
select subdate(i, j) from t;
select subdate(i, concat(j)) from t;
select subdate(i, j / 100) from t;
```

Reply:
Good suggestion to use more complex expressions. Added in a6714fe79dc392d65e9160048b857f21e663db9a